src/components/Intro.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from 'react';
+import styled from 'styled-components'
+
+const P = styled.p`font-size:1.5rem;`
```

Comment:
줄바꿈과 들여쓰기를 보기 좋게 바꿔주세요~
```suggestion
const P = styled.p`
  font-size:1.5rem;
`
```